SustainabilityOnMars/GuidedNotebooks/pets-in-space/challenge-0.ipynb

```diff
@@ -0,0 +1,137 @@
+      "text/html": [
+       "\n",
+       "        <iframe\n",
+       "            width=\"400\"\n",
+       "            height=\"300\"\n",
+       "            src=\"https://www.youtube.com/embed/P6KNkb7xVjk\"\n",
+       "            frameborder=\"0\"\n",
+       "            allowfullscreen\n",
+       "        ></iframe>\n",
+       "        "
+      ],
+      "text/plain": [
+       "<IPython.lib.display.YouTubeVideo at 0x7f81ddcc7190>"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "YouTubeVideo('P6KNkb7xVjk') "
```

Comment:
I think you did this on purpose so they would have the video without having to run the first cell but note I didn't see the video when I first opened my notebook in the CallystoHub. If you want you can just move this line onto the above code block.